pyiron/lammps/base.py

```diff
@@ -659,7 +659,7 @@ def calc_md(
             delta_temp=delta_temp,
             delta_press=delta_press,
             job_name=self.job_name,
-            rotation_matrix=self._prism.R
+            rotation_matrix=rotation_matrix
```

Comment:
nit: pep8 spaces around =

Reply:
It is a function call, so: 
`PEP 8: E251 unexpected spaces around keyword / parameter equals`